templates/actor/parts/actor-arts.html

```diff
@@ -22,6 +22,7 @@
               class="flexrow fontSize10 rollable"
               name="data.arts.techniques.{{key}}.label"
               data-roll="spont"
+              data-bonus-active-effects="{{ ../data.bonuses.arts.spellCasting }}"7
```

Comment:
Pretty sure the "7" at the end of the line 25 is a typo ;)

Reply:
Done!

Reply:
Regression fixed, too